metricbeat/module/vsphere/virtualmachine/_meta/fields.yml

```diff
@@ -97,9 +97,10 @@
     - name: snapshot
       type: group
       fields:
-        - name: info
+        - name: info.*
           type: object
-          description: Deatils of the snapshots of this virtualmachine.
+          object_type: keyword
```

Comment:
Is adding this helpful. From what we discussed it anyhow takes dynamic mapping ?

Reply:
Yes, this templates might affect future changes of integration.